.env.template

```diff
@@ -11,4 +11,14 @@ LEANSERVER_WORKSPACE=
 
 # Leave empty to use CPU count as max.
 LEANSERVER_MAX_REPLS=
-LEANSERVER_MAX_CONCURRENT_REQUESTS=
\ No newline at end of file
+LEANSERVER_MAX_CONCURRENT_REQUESTS=
+
+# Thresholds on CPU and memory beyond which service becomes unhealthy
+LEANSERVER_HEALTHCHECK_CPU_USAGE_THRESHOLD=
```

Comment:
CPU is often very quickly getting to 100%, it probably won't be used to make the service unhealthy.